src/luascript.cpp

```diff
@@ -10467,16 +10467,20 @@ int LuaScriptInterface::luaHouseSetAccessList(lua_State* L)
 int LuaScriptInterface::luaItemTypeCreate(lua_State* L)
 {
 	// ItemType(id or name)
-	uint32_t id;
+	uint32_t id = 0;
```

Comment:
Why? You are setting the id value on both branches on the next lines, `id` will guaranteedly have a value.